### PR TITLE
Task s2 environment setup

### PR DESCRIPTION
**Pull request by karimkefi**
file structure and autoload set up

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<?php require_once __DIR__ . '/vendor/autoload.php'; ?>
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>eCommerce Page</title>
```
Comment (JamesCullen2) [inline index.php]: should we call this by name of store...Totally Pawesome?

File: src/app/categories.php [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<?php
+/**
```
Comment (JamesCullen2) [inline src/app/categories.php]: could we get rid of this multi line comment ?

File: src/app/products.php [hunk 3]
```diff
@@ -0,0 +1,7 @@
+<?php
+/**
```
Comment (JamesCullen2) [inline src/app/products.php]: could we get rid of this multi line comment ?

File: src/test/classes/category.php [hunk 4]
```diff
@@ -0,0 +1,7 @@
+<?php
+/**
```
Comment (JamesCullen2) [inline src/test/classes/category.php:2]: could we get rid of this multi line comment ?

File: src/test/classes/image.php [hunk 5]
```diff
@@ -0,0 +1,7 @@
+<?php
+/**
```
Comment (JamesCullen2) [inline src/test/classes/image.php:2]: could we get rid of this multi line comment ?

File: src/test/classes/product.php [hunk 6]
```diff
@@ -0,0 +1,7 @@
+<?php
+/**
```
Comment (JamesCullen2) [inline src/test/classes/product.php:2]: could we get rid of this multi line comment ?

**Review by JamesCullen2 — CHANGES_REQUESTED** [review]
All good gents, just a few extremely minor changes

**Review by kyam-mayden — APPROVED** [review]

**Pull request merged**